pkg/apis/jaegertracing/v1/jaeger_types.go

```diff
@@ -183,6 +183,10 @@ type JaegerCommonSpec struct {
 
 	// +optional
 	ServiceAccount string `json:"serviceAccount,omitempty"`
+
+	// +optional
+	// +listType=set
+	InitContainers []v1.Container `json:"initContainers,omitempty"`
```

Comment:
Does this have to be exposed?

Reply:
How do we hide it? we can't call it initContainers, but `json:"-"` and removing `+` tags I guess, I have not tried.

Reply:
Moved to storage.GetInitContainers(a.jaeger, commonSpec) - what do you think?

Reply:
I think this is solved